src/py_eddy_tracker/dataset/grid.py

```diff
@@ -1327,7 +1327,7 @@ def kernel_lanczos(self, lat, wave_length, order=1):
         return self.finalize_kernel(kernel, order, half_x_pt, half_y_pt)
 
     def kernel_bessel(self, lat, wave_length, order=1):
-        """wave_length in km
+        """wavelength in km
```

Comment:
n this case we talked about variable which are called "wave_length", i don't know the best soltuion, maybe  add docstring for argument like :
:param float wave_length: in km